localstack/services/ses/provider.py

```diff
@@ -128,7 +128,10 @@ class SesServiceApiResource:
     - GET param `email`: filter for `source` field in SES message
     """
 
-    def on_get(self, request):
+    def on_get(self, request, message_id=None):
+        if message_id is not None:
+            return EMAILS[message_id]
```

Comment:
Let's keep the response schema the same as before, whether or not the `message_id` filter is specified.

```
{
  "messages": [
    {<the message>}
  ]
}
```